app/views/certificates/pathways/show.html.erb

```diff
@@ -94,7 +94,16 @@
       </div>
 
       <div class="govuk-grid-column-one-third">
-        <%= render PathwayEnrolComponent.new(programme: @programme, pathway: @pathway, current_user: @current_user) %>
+        <%= render AsideComponent.new(title: 'How to enrol') do |component| %>
+          <% component.with_body do %>
+            <ol class="govuk-body-s">
+              <% @pathway.enrol_copy.each do |line| %>
+                <li><%= line.html_safe %></li>
+              <% end %>
+            </ol>
+            <%= render EnrolmentConfirmationComponent.new(current_user:, programme: Programme.primary_certificate) %>
```

Comment:
I don't think this should be hard coded to programme primary, as programme is variable in this file so it can be used by multiple programmes